### PR TITLE
chore(): reduce redundant code size

### PR DESCRIPTION
**Pull request by asturur**
## Motivation

Removing redundant code in non performance critical spot.
This code could perform worse than the one before but this isn't really a performance impacting area

Just less bulky code in general

### CODE REVIEW

File: src/canvas/Canvas.ts [hunk 1]
```diff
@@ -1585,10 +1585,6 @@ export class Canvas extends SelectableCanvas {
     return true;
   }
 
-  exitTextEditing() {
-    this.textEditingManager.exitTextEditing();
-  }
```
Comment (asturur) [inline src/canvas/Canvas.ts:1590]: this doesn't seem used by anyone and wasn't in jsdocs either.

File: src/shapes/IText/ITextKeyBehavior.ts [hunk 2]
```diff
@@ -64,43 +64,41 @@ export abstract class ITextKeyBehavior<
...
-      this.onCompositionStart.bind(this)
-    );
-    this.hiddenTextarea.addEventListener(
-      'compositionupdate',
-      this.onCompositionUpdate.bind(this)
-    );
-    this.hiddenTextarea.addEventListener(
-      'compositionend',
-      this.onCompositionEnd.bind(this)
+    textarea.style.cssText = `position: absolute; top: ${top}; left: ${left}; z-index: -999; opacity: 0; width: 1px; height: 1px; font-size: 1px; padding-top: ${fontSize};`;
```
Comment (ShaMan123) [inline src/shapes/IText/ITextKeyBehavior.ts:82]: I think we should setStyles here

File: src/shapes/IText/ITextKeyBehavior.ts [hunk 3]
```diff
@@ -64,43 +64,41 @@ export abstract class ITextKeyBehavior<
...
       getFabricDocument();
-    this.hiddenTextarea = doc.createElement('textarea');
-    this.hiddenTextarea.setAttribute('autocapitalize', 'off');
-    this.hiddenTextarea.setAttribute('autocorrect', 'off');
-    this.hiddenTextarea.setAttribute('autocomplete', 'off');
-    this.hiddenTextarea.setAttribute('spellcheck', 'false');
-    this.hiddenTextarea.setAttribute('data-fabric', 'textarea');
-    this.hiddenTextarea.setAttribute('wrap', 'off');
-    const style = this._calcTextareaPosition();
+    const attributes = {
```
Comment (ShaMan123) [inline src/shapes/IText/ITextKeyBehavior.ts:67]: don't you prefer to inline the arg if it is used only once?

**Review by ShaMan123 — DISMISSED** [review]

File: src/shapes/IText/ITextKeyBehavior.ts (hunk 3 above)
Comment (asturur) [reply src/shapes/IText/ITextKeyBehavior.ts:67]: sometimes i have some readibility concenrn but here seems fine to do so.

**Pull request merged**